src/stylesheets/components/_banner.scss

```diff
@@ -1,206 +1,244 @@
 .usa-banner {
   @include typeset($theme-font-family-banner);
-  @include u-bg('base-lightest');
-  @include u-padding-bottom(1);
-  @include u-position('relative');
+  background-color: color('base-lightest');
 
-  @include at-media('mobile-lg') {
-    @include u-font-size($theme-font-family-banner, $theme-font-size-h6);
-    @include u-padding-bottom(0);
+  @include at-media('tablet') {
+    font-size: font-size($theme-font-family-banner, '3xs');
+    padding-bottom: units(0);
   }
 }
 
 .usa-banner-content {
-  @include u-bg('transparent');
-  @include u-padding-bottom(2px);
-  @include u-padding-top(2.5);
-  @include u-padding-x($site-margins-mobile);
-  @include u-width('full');
+  @include grid-container($theme-site-max-width);
+  background-color: color(transparent);
+  font-size: font-size($theme-font-family-banner, 4);
+  overflow: hidden;
+  padding-bottom: units(2);
+  padding-top: units(0.5);
+  padding-right: units($theme-site-margins-mobile);
+  padding-left: units($theme-site-margins-mobile - 1);
```

Comment:
Alphabetize `padding-*` properties.

Reply:
Is there a way to check for this with linting?

Reply:
Maybe https://www.npmjs.com/package/stylelint-order#list-of-rules? Would rather the machine do this check than us humans. Though I don't know how it handles includes. Something to try out.